cmd/zdb/zdb.c

```diff
@@ -4360,7 +4360,7 @@ dump_l2arc_log_blkptr(l2arc_log_blkptr_t lbps)
 }
 
 static void
-dump_l2arc_log_blocks(int fd, l2arc_dev_hdr_phys_t l2dhdr,
+dump_l2arc_log_blocks(int fd, l2arc_dev_hdr_phys_t *l2dhdr,
```

Comment:
I'd add const here and in dump_l2arc_log_entries() just for clarity.

Reply:
Good idea. Also, nice catch on `dump_l2arc_log_blkptr()`. Coverity failed to report that one. Changing it to use a constant pointer means we can cleanup the weird `(&lbps)->lbp_prop`.

I have pushed a revision incorporating these changes.